CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## v6.2.3

```

Comment:
i think there is already a 6.2.3, try merging with latest

Reply:
merged and update to 6.2.5